components/speakers/keynote-speakers-data.js

```diff
@@ -1,27 +1,27 @@
 export const KEYNOTESPEAKERS = [
     {
-      name: "Anthony Shaw",
```

Comment:
these are for last year, it would be nice that we maintain them as they are used for the [2023 site ](https://ug.pycon.org/2023) so for 2024, would appear on this [2024 site](https://ug.pycon.org/2024)